diff --git a/node_modules/spacetimedb/dist/index.browser.mjs b/node_modules/spacetimedb/dist/index.browser.mjs
index e290522..2ece975 100644
--- a/node_modules/spacetimedb/dist/index.browser.mjs
+++ b/node_modules/spacetimedb/dist/index.browser.mjs
@@ -2232,12 +2232,12 @@ var ArrayBuilder = class extends TypeBuilder {
   }
   default(value) {
     return new ArrayColumnBuilder(
-      this.element,
+      this,
       set(defaultMetadata, { defaultValue: value })
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/index.cjs b/node_modules/spacetimedb/dist/index.cjs
index 04b9bed..f03dcf2 100644
--- a/node_modules/spacetimedb/dist/index.cjs
+++ b/node_modules/spacetimedb/dist/index.cjs
@@ -2234,12 +2234,12 @@ var ArrayBuilder = class extends TypeBuilder {
   }
   default(value) {
     return new ArrayColumnBuilder(
-      this.element,
+      this,
       set(defaultMetadata, { defaultValue: value })
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/index.mjs b/node_modules/spacetimedb/dist/index.mjs
index 282a534..fffa4a8 100644
--- a/node_modules/spacetimedb/dist/index.mjs
+++ b/node_modules/spacetimedb/dist/index.mjs
@@ -2237,7 +2237,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/sdk/index.browser.mjs b/node_modules/spacetimedb/dist/sdk/index.browser.mjs
index 5998c39..d4d6488 100644
--- a/node_modules/spacetimedb/dist/sdk/index.browser.mjs
+++ b/node_modules/spacetimedb/dist/sdk/index.browser.mjs
@@ -1941,7 +1941,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/sdk/index.cjs b/node_modules/spacetimedb/dist/sdk/index.cjs
index eb6ebf8..e0d15c0 100644
--- a/node_modules/spacetimedb/dist/sdk/index.cjs
+++ b/node_modules/spacetimedb/dist/sdk/index.cjs
@@ -1943,7 +1943,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/sdk/index.mjs b/node_modules/spacetimedb/dist/sdk/index.mjs
index 8ae5bfd..27784b4 100644
--- a/node_modules/spacetimedb/dist/sdk/index.mjs
+++ b/node_modules/spacetimedb/dist/sdk/index.mjs
@@ -1941,7 +1941,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/server/index.cjs b/node_modules/spacetimedb/dist/server/index.cjs
index df95d31..90095c4 100644
--- a/node_modules/spacetimedb/dist/server/index.cjs
+++ b/node_modules/spacetimedb/dist/server/index.cjs
@@ -2358,7 +2358,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
diff --git a/node_modules/spacetimedb/dist/server/index.mjs b/node_modules/spacetimedb/dist/server/index.mjs
index 935cb22..a02aee1 100644
--- a/node_modules/spacetimedb/dist/server/index.mjs
+++ b/node_modules/spacetimedb/dist/server/index.mjs
@@ -2336,7 +2336,7 @@ var ArrayBuilder = class extends TypeBuilder {
     );
   }
   name(name) {
-    return new ArrayColumnBuilder(this.element, set(defaultMetadata, { name }));
+    return new ArrayColumnBuilder(this, set(defaultMetadata, { name }));
   }
 };
 var ByteArrayBuilder = class extends TypeBuilder {
